project_resource_calendar/tests/common.py

```diff
@@ -8,7 +8,12 @@ class TestCalendarEventCommon(common.TransactionCase):
 
     def setUp(self):
         super(TestCalendarEventCommon, self).setUp()
-
+        # Initialization of param values
+        room_name = 'Testing_Room_type_2'
+        resource_type_name = 'room'
+        instrument_name = 'Testing_Instrument_type_2'
```

Comment:
why the name value is separated by  : '_'  room or instrument name is a string so you can separate words by space.

Reply:
@rimbendhaou : use to remove the wildcard but changed it